### PR TITLE
Uicleanup

### PR DESCRIPTION
**Pull request by harrionparrix**
Change Name from Sandhi to OCR-Editor

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@
 
 1. If you have the error while running mongodb and the error says mongodb.config is not found then change the name of the file mongod.config to mongodb.config in the etc directory. This usually happens if you install mongodb via **sudo apt-get instal**. L
 2. Now after installing mongodb successful, run mongo on terminal and note down the port number, for instance if the address is **mongodb://127.0.0.1:27017** the port number is 27017. By default this should be the port number on which mongodb is running but its okay if the port number is different. 
-3. Now copy this port number and add to his address to “mongodb://localhost:[Port Number]/sandhi-books”, without the brackets. Now go to sandhiserver/config/appconfig.py and paste this address after ‘host’.
+3. Now copy this port number and add to his address to “mongodb://localhost:[Port Number]/OCR-Editor-books”, without the brackets. Now go to server/config/appconfig.py and paste this address after ‘host’.
```
Comment (venkatapathy) [inline README.md:79]: it should be just ocr-editor

File: pycodes/pdf_to_txt_tesseract_ocr.py [hunk 2]
```diff
@@ -11,16 +11,16 @@
...
+OCR_EDITOR_LIB_INPUT_DIR = '/home/ruralivrs/ocr-editor/input_books/'
+OCR_EDITOR_LIB_OUTPUT_DIR = '/home/ruralivrs/ocr-editor/output_books/'
 
 
 sys.path.append('../server/mgodb')
 from models import Book
 from mongoengine import connect
 
 relevant_path = input(
-    "Give path to the folder where pdfs. Enter for default option: [~/Documents/ocr/sandhi/input_books]"
```
Comment (venkatapathy) [inline pycodes/pdf_to_txt_tesseract_ocr.py:23]: can we make [~/Documents/ocr/sandhi/input_books] into [~/Documents/ocr/sandhi/input_books]? @harrionparrix

**Review by venkatapathy — CHANGES_REQUESTED** [review]

**Review by harrionparrix — COMMENTED** [review]
ocr-editor in place of ocr-editor-books

**Review by venkatapathy — CHANGES_REQUESTED** [review]
This should not be pushed to the master yet. I have it in the branch cleanup and uicleanup. Remove this as part of this pull request, please.

File: frontend/client/src/components/PageView/PageView.css [hunk 3]
```diff
@@ -0,0 +1,37 @@
+.background{
```
Comment (venkatapathy) [inline frontend/client/src/components/PageView/PageView.css:1]: the UI changes can be skipped in this pull request

File: frontend/client/src/components/PageView/PageView.tsx [hunk 4]
```diff
@@ -1,12 +1,14 @@
 import ToolBar from "./ToolBar";
```
Comment (venkatapathy) [inline frontend/client/src/components/PageView/PageView.tsx:1]: Again UI changes can be skipped. sandhi to OCR-editor changes alone can be added

**Review by venkatapathy — CHANGES_REQUESTED** [review]
@harrionparrix please check my review comments and make appropriate changes

**Pull request closed without merging**